BaseTools/Conf/build_rule.template

```diff
@@ -768,29 +768,16 @@
         GenFw -o $(OUTPUT_DIR)(+)$(MODULE_NAME)hii.rc -g $(MODULE_GUID) --hiibinpackage $(HII_BINARY_PACKAGES)
 
 # MU_CHANGE Start
-[Resource-File]
+[Resource-File.UEFI_HII]

```

Comment:
does this name actually matter?  

If not was your reason for adding .UEFI_HII to indicate that this rule translates from Resource into hpk for use by HII rule above?

Reply:
It doesn't matter. The reason why I changed is just for naming consistency with other rules as you said (e.g: `Visual-Form-Representation-File.UEFI_HII`, `Image-Definition-File.UEFI_HII`) that also convert other types of input files into `.hpk` for the the `Hii-Binary-Package.UEFI_HII` to process them.

I can comment in the commit message the rationale for the change to make it more clear or I can also drop that bit if you prefer.

Reply:
I just wanted to confirm understanding i think it makes sense as you have it.